config/suit/templates/nrf54h20/default/v1/app_recovery_envelope_direct.yaml.jinja2

```diff
@@ -1,10 +1,10 @@
 {%- set component_index = 0 %}
 {%- set component_list = [] %}
 {%- set dependencies_list = [] %}
-{%- set mpi_app_recovery_vendor_name = application['config']['CONFIG_SUIT_MPI_APP_RECOVERY_VENDOR_NAME']|default('nordicsemi.com') %}
```

Comment:
It seems similar changes are missing from `app_recovery_local_envelope.yaml.jinja2` and `app_recovery_local_envelope.yaml.jinja2`